uisdk/src/main/res/values/integers.xml

```diff
@@ -19,4 +19,8 @@
 
     <!-- Map Zoom Levels -->
     <item name="kh_uisdk_map_zoom_max" format="float" type="dimen">20.0</item>
+
+    <!-- Query List View-->
+    <integer name="kh_uisdk_query_list_view_default_screen_percentage">70</integer>
```

Comment:
Can you please check with olivier what would be the ideal numbers? 
I suggest the default to be 40 and the max screen percentage to be 75 or 80.